### PR TITLE
Training example

### PR DESCRIPTION
**Pull request by jburel**
# What this PR does

simplify call to load OriginalFile

# Testing this PR

Check that the example is still passing
# Related reading

See https://github.com/ome/ome-documentation/pull/2022

cc @mtbc

### CODE REVIEW

**Comment by mtbc** [comment]
Looks promising! Let's see what [OMERO-training](https://merge-ci.openmicroscopy.org/jenkins/job/OMERO-training/) thinks.

File: examples/Training/java/src/training/WriteData.java [hunk 1]
```diff
@@ -315,12 +315,13 @@ private void loadAnnotationsLinkedToImage()
...
             FileOutputStream stream = new FileOutputStream(file);
             OriginalFile of;
+            IQueryPrx svc = gateway.getQueryService(ctx);
             while (j.hasNext()) {
                 annotation = j.next();
                 if (annotation instanceof FileAnnotation && index == 0) {
                     fa = new FileAnnotationData((FileAnnotation) annotation);
                     //The id of te original file
-                    of = getOriginalFile(fa.getFileID());
+                    of = svc.get("OriginalFile", omero.rtypes.rlong(fa.getFileID());
```
Comment (mtbc) [inline examples/Training/java/src/training/WriteData.java]:
```suggestion
                    of = svc.get("OriginalFile", omero.rtypes.rlong(fa.getFileID()));
```

**Comment by mtbc** [comment]
It's alarming that Travis passed.

File: examples/Training/java/src/training/WriteData.java [hunk 2]
```diff
@@ -315,12 +315,13 @@ private void loadAnnotationsLinkedToImage()
...
             FileOutputStream stream = new FileOutputStream(file);
             OriginalFile of;
+            IQueryPrx svc = gateway.getQueryService(ctx);
             while (j.hasNext()) {
                 annotation = j.next();
                 if (annotation instanceof FileAnnotation && index == 0) {
                     fa = new FileAnnotationData((FileAnnotation) annotation);
                     //The id of te original file
-                    of = getOriginalFile(fa.getFileID());
+                    of = svc.get("OriginalFile", omero.rtypes.rlong(fa.getFileID()));
```
Comment (mtbc) [inline examples/Training/java/src/training/WriteData.java]:
```suggestion
                    of = svc.get("OriginalFile", fa.getFileID());
```

**Comment by joshmoore** [comment]
What's the status here?

**Comment by jburel** [comment]
This works but we could not validate in CI due to Python work.

**Comment by jburel** [comment]
https://merge-ci.openmicroscopy.org/jenkins/job/OMERO-training/198/
So we can merge this PR

**Comment by jburel** [comment]
Doc PR updated cc @mtbc

**Pull request merged**